taskcluster/ci/version-bump/kind.yml

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: fenix_taskgraph.loader.multi_dep:loader
+
+transforms:
+    - fenix_taskgraph.transforms.multi_dep:transforms
+    - fenix_taskgraph.transforms.version_bump:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - github-release
+
+primary-dependency: github-release # FIXME: should we block on pushapk instead or both?
```

Comment:
`push-apk` is a good idea. `github-release` doesn't depend on the state of the tree so both tasks can run at the same time, in my opinion.